cluster-up/cluster/ocp-4.3/provider.sh

```diff
@@ -44,6 +44,13 @@ function up() {
     user=$(echo $auth |awk -F: '{print $1}')
     password=$(echo $auth |awk -F: '{print $2}')
 
+    # If provision test mode is on, use local image
+    if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
+        params=" --container-registry ${container_registry} $params"
+    else
+        params=" --container-registry= $params"
```

Comment:
What does '--container-registry=' alone do?

Reply:
it is mandatory that when you dont want gocli add a prefix like docker (default) / quay
to the image its going to look for,
this is the method for telling it to add a null string as prefix, needed when the image is local (after provision its local)

Reply:
IMHO it's a sign of a smell here that we're confusing the user with an empty string. I think we should rather use an empty string as default value for gocli and then set an external location explicitly instead of what we're doing now: setting the parameter to an empty string. But that's a follow up issue.